spec/facter/facter_spec.rb

```diff
@@ -403,6 +403,20 @@ def mock_collection(method, os_name = nil, error = nil)
       end
     end
 
+    describe '#flush' do
+      it 'sends call to LegacyFacter' do
+        allow(LegacyFacter).to receive(:flush)
```

Comment:
unit test should be separated according to the AAA pattern (Arrange, Act, Assert).

Reply:
Updated. Thanks!